tests/test_fluxFitBoundedField.py

```diff
@@ -36,6 +36,10 @@
 
 DATA_DIR = os.path.join(os.path.split(__file__)[0], "data")
 
+# Note: mosaic's internal ffp object is magnitude-based.
+# Instead, for these tests, we just divide this out when comparing.
+referenceFlux = 1e23 * 10**(48.6 / -2.5) * 1e9
```

Comment:
Does this need a #TODO or is this just for the internal tests and meas_mosaic won't be changed even when Calib is retired?

Reply:
No TODO here: mosaic's internal format will not change.